.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Question
+about: A question which is not a bug report but sth which may need to be addressed.
```

Comment:
What is "sth" meant to be?

Reply:
Okay I was to lazy to type... I was referring to anything which is not a bug report, feature request or backport request.